db/migrate/20160726061957_turn_default_value_to_lookup_value.rb

```diff
@@ -0,0 +1,25 @@
+class TurnDefaultValueToLookupValue < ActiveRecord::Migration
+  def up
+    LookupValue.transaction do
+      LookupKey.unscoped.find_each do |key|
+        LookupValue.create(:match => 'default', :value => key.default_value, :lookup_key_id => key.id, :use_puppet_default => key.use_puppet_default)
+      end
+    end
```

Comment:
If you want this to run extremely quickly you can do it by building the SQL yourself. This will execute in practically no time even for a large db (I tried it with 5k values and it took less then 1 second). However the down side is it's much less legible and more prone to errors, so I'm not sure it's worth the effort for shaving a minute from the migration time. Here is the code that should generate valid sql for all adapters:

```ruby
time = Time.now
abc = ActiveRecord::Base.connection
values = LookupKey.unscoped.map{ |key| "(#{abc.quote('default')}, #{abc.quote(key.default_value)}, #{abc.quote(key.id)}, #{abc.quote(key.use_puppet_default)}, #{abc.quote(time)}, #{abc.quote(time)})"}
sql = "INSERT INTO #{abc.quote_table_name(LookupValue.table_name)} (#{[abc.quote_column_name('match'),abc.quote_column_name('value'),abc.quote_column_name('lookup_key_id'),abc.quote_column_name('use_puppet_default'),abc.quote_column_name('created_at'),abc.quote_column_name('updated_at')].join(',')}) VALUES #{values.join(',')}"
abc.execute(sql)
```